### ISSUE TITLE
Login loop

### ISSUE THREAD

**Issue by Iss-in** [fluix-dev/sway-borders#47]
Currently for those who dont use a login manager and run sway from tty using autologin, any error in config ( ie. a typo in image file name ) is making sway endlessly loop from tty. Cant even access other tty as you are switched back to sway tty immediately. 
currently only solution is to access the config file using bootable iso and correct the config.
Is is possible to just ignore the shader if the file dont exist ?

**Comment by fluix-dev** [comment]
> any error in config ( ie. a typo in image file name ) is making sway endlessly loop from tty

Does this apply to other configuration options (namely those present in Sway upstream)?

**Comment by Iss-in** [comment]
no, at best you only get that red error bar telling about the wrong syntax in that line
